hack/common

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+if [ -n "${DEBUG:-}" ]; then
+    set -x
+fi
+
+alias oc=${OC:-oc}
+
+repo_dir="$(dirname $0)/.."
+ELASTICSEARCH_OP_REPO=${ELASTICSEARCH_OP_REPO:-${repo_dir}/../elasticsearch-operator}
```

Comment:
so this means `ELASTICSEARCH_OP_REPO=../../elasticsearch-operator`?

Reply:
I was trying to achieve an absolute path to the repo assumed to be parallel to this one.